CAP/gap/WrapperCategory.gi

```diff
@@ -55,6 +55,13 @@ InstallOtherMethodForCompilerForCAP( AsMorphismInWrapperCategory,
     #% CAP_JIT_DROP_NEXT_STATEMENT
     CAP_INTERNAL_ASSERT_IS_MORPHISM_OF_CATEGORY( morphism, ModelingCategory( D ), [ "the morphism given to AsMorphismInWrapperCategory" ] );
     
+    #% CAP_JIT_DROP_NEXT_STATEMENT
+    if not IsEqualForObjects( ModelingCategory( D ), ObjectDatum( D, source ), Source( morphism ) ) then
+        Error( "ObjectDatum( source ) and Source( morphism ) do not coincide\n" );
+    elif not IsEqualForObjects( ModelingCategory( D ), ObjectDatum( D, range ), Range( morphism ) ) then
```

Comment:
These checks are semantically independent, so I suggest two `if ... fi;` instead of using `elif`.

Reply:
Done.